library/cipher_wrap.c

```diff
@@ -210,8 +210,9 @@ static int aes_setkey_enc_wrap( void *ctx, const unsigned char *key,
 
 static void * aes_ctx_alloc( void )
 {
-    mbedtls_aes_context *aes = mbedtls_calloc( 1, sizeof( mbedtls_aes_context ) );
+    mbedtls_aes_context *aes;
 
+    aes = mbedtls_calloc( 1, sizeof( mbedtls_aes_context ) );
```

Comment:
I suggest removing this change, this seems to be more of a style issue that something necessary for parameter validation.